lib/catalog/api/minion/order.rb

```diff
@@ -0,0 +1,51 @@
+require 'catalog-api-client'
+require "rest-client"
+require "catalog/api/minion/base"
+require "uri"
+
+module Catalog
+  module Api
+    class Minion
+      class Order < Base
+        include Logging
+
+        def base_name
+          "Catalog API Order Minion"
+        end
+
+        def queue_name
+          "platform.topological-inventory.task-output-stream".freeze
+        end
+
+        def persist_ref
+          "catalog-api-order-minion".freeze
+        end
+
+        def perform(message)
+          jobtype = message.message
+          payload = message.payload
+          payload_params = {:payload =>  message.payload, :message => jobtype}
+
+          logger.info("#{jobtype}: #{payload}")
+          response = post_internal_notify(payload, payload_params)
+          logger.info("#{response}")
+        end
+
+        private
+
+        def post_internal_notify(payload, payload_params)
+          RestClient.post(internal_notify_url(payload['task_id']), payload_params, identity_headers(persist_ref))
+        end
+
+        def internal_notify_url(task_id)
+          config = ::CatalogApiClient.configure
```

Comment:
See my changes here: https://github.com/ManageIQ/catalog-api-minion/pull/10/files#diff-68e153a4bcfe0f001ff02ef7f54a836b which are required to statically define the catalog-ci url until we can get an e2e-deploy repo setup which will dynamically override `config = ::CatalogApiClient.configure` to have `config.host` return the correct host and `config.port` return the correct port.